app/controllers/cogy/application_controller.rb

```diff
@@ -1,6 +1,6 @@
 module Cogy
   class ApplicationController < ActionController::Base
     # @todo https://github.com/skroutz/cogy/issues/43
-    skip_before_action :verify_authenticity_token
+    protect_from_forgery unless: -> { request.format.json? }
```

Comment:
This is not equivalent to the previous behavior. This will probably break the cogy bundle since it does [_not_ set the content-type to JSON](https://github.com/skroutz/cogy-bundle/blob/master/commands/cogy#L12).

We should be fine by just removing the `skip_before_action` call.